roles/kubernetes/client/tasks/main.yml

```diff
@@ -38,50 +21,33 @@
   delegate_to: localhost
   become: no
   run_once: yes
-  when: kubeconfig_localhost|default(false)
+  when: kubeconfig_localhost
 
 - name: Wait for k8s apiserver
   wait_for:
     host: "{{ kube_apiserver_access_address }}"
     port: "{{ kube_apiserver_port }}"
     timeout: 180
 
-# NOTE(mattymo): Please forgive this workaround
```

Comment:
@mattymo do you remember why this workaround was necessary?

Reply:
ping @mattymo